src/app/core/data/data.service.ts

```diff
@@ -90,4 +101,15 @@ export abstract class DataService<TNormalized extends CacheableObject, TDomain>
     // return this.rdbService.buildSingle(href));
   }
 
+  findEmbedded<TEmbeddedNormalized extends CacheableObject, TEmbeddedDomain>(
```

Comment:
Sorry, I don't understand why the /discover/browses/dateissued endpoint come in this discussion :( 
I can say that the right endpoints to retrieve all the collections (paginated) within a community is like
http://dspace7.4science.it/dspace-spring-rest/api/core/communities/7669c72a-3f2a-451f-a3b9-9210e7a4c02f/collections
another endpoint could be added if needed (I don't see a reason for that honestly) but should be like
/collections/search/findByCommunity?scope=:uuid